src/SprykerSdk/Spryk/Model/Spryk/Definition/Argument/Resolver/ArgumentResolver.php

```diff
@@ -279,4 +280,24 @@ protected function choseArgumentValue(string $argument, string $sprykName, array
 
         return $this->style->askQuestion($question);
     }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface $argumentCollection
+     *
+     * @return \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface
+     */
+    protected function resolveOrganizationAndModuleRootPath(ArgumentCollectionInterface $argumentCollection): ArgumentCollectionInterface
+    {
+        if (!$argumentCollection->hasArgument('organization')) {
+            return $argumentCollection;
+        }
+
+        if (!in_array($argumentCollection->getArgument('organization')->getValue(), ['Spryker', 'SprykerShop', 'Pyz'])) {
```

Comment:
Does it make sense to use config constants here?

```
$config[KernelConstants::PROJECT_NAMESPACES] = [
    'Pyz',
];
$config[KernelConstants::CORE_NAMESPACES] = [
    'SprykerShop',
    'SprykerEco',
    'Spryker',
    'SprykerSdk',
];
```

And to think: maybe it's better to set `spryker/` and `Bundles/` in case of 'Spryker'/ 'SprykerShop' and have empty by default?

Reply:
> Does it make sense to use config constants here?

Have no idea how to nicely get only Spryker and SprykerShop from CORE_NAMESPACES without specifying them as exact strings

> And to think: maybe it's better to set spryker/ and Bundles/ in case of 'Spryker'/ 'SprykerShop' and have empty by default?

why not, changed

Reply:
> And to think: maybe it's better to set spryker/ and Bundles/ in case of 'Spryker'/ 'SprykerShop' and have empty by default?

Unfortunately, i reverted this change, bcz otherwise we need to add more specific logic here(split if for Pyz and for 'Spryker', 'SprykerShop', and code duplication)